apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V112__Drop_Support_Medicare_BeneficiaryId_History.sql

```diff
@@ -0,0 +1,15 @@
+-- BFD-2629
+-- Drop_Support_Medicare_BeneficiaryId_History
+-- flyway migration to drop support for;
+--   - medicare_beneficiaryid_history
+--   - medicare_beneficiaryid_history_invalid_beneficiaries
+--
+-- These tables were created as a one-time CCW data drop containing
+-- historical MBI data; historical MBI data (i.e., MBI_NUM) can be
+-- used to derive a bene_id (beneficiary) using old(er) data.
+-- 
+-- Current BFD functionality for looking up bene_id using historical
+-- MBI_NUM is now handled by the beneficiaries_history table.
+--
+drop table if exists medicare_beneficiaryid_history cascade;
+drop table if exists medicare_beneficiaryid_history_invalid_beneficiaries cascade;
```

Comment:
You had a temp backup table too, right? Is that being cleaned up OOB?

Reply:
The temp table was created OOB and was intended to be deleted OOB; however, using something like 'if exists' and delete it part of this flyway script is amenable. Normally we've taken the tack of 'if it is created by flyway, we delete via flyway'; will add to the script here since it's pretty harmless if the table doesn't exist.